src/audiomixerboard.cpp

```diff
@@ -1508,6 +1544,11 @@ void CAudioMixerBoard::SetChannelLevels ( const CVector<uint16_t>& vecChannelLev
     {
         if ( vecpChanFader[iChId]->IsVisible() && ( i < iNumChannelLevels ) )
         {
+            // compute exponential moving average
+            float alpha = AUTO_FADER_ADJUST_ALPHA;
```

Comment:
I think either you or I missed one of a minor comment :).  Can you just use `AUTO_FADER_ADJUST_ALPHA`, rather than creating a variable to hold a constant, please.

Reply:
I will change that in the next commit. This answers my question above:

> Or would you prefer removing alpha completely and using AUTO_FADER_ADJUST_ALPHA instead?